ExampleMod/Content/Items/Armor/ExampleBreastplate.cs

```diff
@@ -32,7 +32,7 @@ public class ExampleBreastplate : ModItem
 		}
 
 		public override void AddRecipes() {
-			CreateRecipe().AddIngredient<ExampleItem>(60) // this item needs 60 ExampleItem's to craft
+			CreateRecipe().AddIngredient<ExampleItem>(60) // this item needs 60 ExampleItems to craft
 				.AddTile<Tiles.Furniture.ExampleWorkbench>() // craft the item at an ExampleWorkbench
 				.Register();
 		}
```

Comment:
All AddRecipes() overrides in files that don't particularly showcase anything related to recipes should require 1 ExampleItem to craft, and also this notice at the top:
```cs
// Please see Content/ExampleRecipes.cs for a detailed explanation of recipe creation.
public override void AddRecipes() {
```
There's no need for comments besides that one.